README.md

```diff
@@ -1 +1,48 @@
-# Portfolio
\ No newline at end of file
+
+# Microverse Portfolio Project
+> Portfolio project created using Figma design.
```

Comment:
- [OPTIONAL]  Please, add a meaningful  `Project description. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯